### PR TITLE
fix: add group about

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/7525

required https://github.com/fossasia/open-event-server/pull/8088
Screenshot from 2021-07-08 17-49-33
Screenshot from 2021-07-08 17-48-57

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks.

Please move it below the group name in both views/pages.

**Comment by maze-runnar** [comment]
Screenshot from 2021-07-14 16-20-15
Screenshot from 2021-07-14 16-18-50
Screenshot from 2021-07-14 16-25-46

**Comment by ranjitkshah** [comment]
@SundaramDubey **LOKI 06**  😄

**Review by MeghalBisht — APPROVED** [review]

File: app/templates/components/group-card.hbs [hunk 1]
```diff
@@ -26,15 +29,6 @@
...
       </div>
-      <div class="meta">
-        <span class="time">
-            {{general-date @group.modifieddAt 'date-time-short'}}
-        </span>
-        <br>
-        {{#each @group.events as |event|}}
-          <a href="{{event.url}}" target="_blank" rel="noopener"  class="ui blue basic compact button">{{event.name}} </a>
-        {{/each}}
-      </div>
```
Comment (MeghalBisht) [inline app/templates/components/group-card.hbs:37]: need to add this back

**Pull request merged**